app/assets/javascripts/application.js

```diff
@@ -16,3 +16,13 @@
 //= require_tree .
 
 $(function(){ $(document).foundation(); });
+
+def reverse_string(string)
```

Comment:
Identifier 'reverse_string' is not in camel case.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'def' is not defined.<br>'reverse_string' is not defined.<br>'string' is not defined.